0002-allow-building-outside-of-the-gcc-tree.patch

```diff
@@ -0,0 +1,124 @@
+From 07084fe8b33f9c7bedbf3c1d0c8df813935268e8 Mon Sep 17 00:00:00 2001
+From: Steve Lhomme <robux4@ycbcr.xyz>
+Date: Tue, 7 Apr 2020 13:56:58 +0200
+Subject: [PATCH 2/2] allow building outside of the gcc tree
```

Comment:
Reorder the patches, making this one (the prerequisite for this building at all) number 1.

Place the patches in a subdir like `patches/libssp/0001-foo-bar.patch`.

Reply:
I can change the names but they correspond to order in my branch. The 0001 is the one I'd like upstreamed. The 0002 will never be. But to use 0001 in this project, we need 0002 first.

Reply:
Well regardless of what order they are in your branch, here in this project, I'd want them in the order and numbering that makes sense over here.

Reply:
I reordered them. In the end the bcrypt one (now 0002) will disappear and will be merged in gcc while out of tree (0001) will probably not and so will remain unchanged.